plugins/woocommerce-blocks/tests/e2e/utils/editor/editor-utils.page.ts

```diff
@@ -162,15 +176,9 @@ export class EditorUtils {
 	async toggleGlobalBlockInserter() {
 		// "Add block" selector is required to make sure performance comparison
 		// doesn't fail on older branches where we still had "Add block" as label.
```

Comment:
This comment seems out of place now, no?

Reply:
Addressed with [c4bd942](https://github.com/woocommerce/woocommerce/pull/44593/commits/c4bd9425cb6e52f6632dac5a630fa6e943b70759)